Frontend/ui-driver/src/Services/Endpoint.purs

```diff
@@ -98,3 +98,6 @@ linkReferralCode dummyString = (getBaseUrl "") <> "/driver/linkReferralCode"
 
 getPerformance :: String -> String
 getPerformance dummyString = (getBaseUrl "") <> "/driver/performance"
+
+otpRide :: String -> String
+otpRide dummyRideOtp = (getBaseUrl "") <> "/driver/otpRide/start"
```

Comment:
What is dummyRide Otp here?

Reply:
for lazy evaluation we need to give some arguments